pidgin/app.py

```diff
@@ -113,9 +112,13 @@ def send_query(query_txt):
     query = {'query': query_txt}
     data = requests.post(api_url, headers={'Authorization': auth}, json=query).json()
 
-    return flask.jsonify(data)
+    no_auth_msg = 'No authentication is provided'
```

Comment:
I think there should be a more robust way to check for an error here than with this string matching. If the status code from the request is 403 that's probably sufficient to raise an auth error, or if the response looked something like `{"error": {"code": 403, "message": "..."}}` that would be ideal